### PR TITLE
Fixing broken links in data-types markdown document

### PR DESCRIPTION
**Pull request by bGrass**
---
name: Brian Grass
about: Usability 
title: Fixing broken links in data-types markdown document

---

# MDS Pull Request

Thank you for your contribution!  Please review our OMF [contributing page](https://github.com/openmobilityfoundation/governance/blob/main/CONTRIBUTING.md) to understand guidelines and policies for participation, and our [Code of Conduct page](https://github.com/openmobilityfoundation/governance/blob/main/CODE_OF_CONDUCT.md).

To avoid complications and help make the Review process as smooth as possible, make sure to:

1. Target [**`dev`**](https://github.com/openmobilityfoundation/mobility-data-specification/tree/dev) branch. Please ensure you are targeting **`dev`**, not **`main`**. 
1. Keep the *"Allow edits from maintainers"* button checked to help us resolve some issues for you.
1. Be ready to resolve any merge conflicts before we approve your Pull Request.
1. Have an up to date profile, per our Github [community profile](https://github.com/openmobilityfoundation/governance/blob/main/CONTRIBUTING.md#community-profile) guildance.

## Explain pull request

My colleagues and I got a little tripped up looking for the vehicle types definitions since some links to that location were broken.  I found some other similar issue on the same page and have tried to address them all.

## Is this a breaking change

No, this is a usability fix to the markdown documents only.

## Impacted Spec

Which spec(s) will this pull request impact?

None

### CODE REVIEW

File: data-types.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ This MDS data types page catalogs the objects (fields, types, requirements, desc
 ## Table of Contents
 
 - [Vehicles](#vehicles)
-  - [Vehicle Types](#vehicle-types)
+  - [Vehicle Types][vehicle-types]
   - [Propulsion Types](#propulsion-types)
   - [Vehicle Status](#vehicle-status)
 - [Events](#events)
-  - [Event Types](#event-times)
```
Comment (schnuerle) [inline data-types.md:12]: The link here is actually correct.  The text should read 'Event Times'.

File: data-types.md [hunk 2]
```diff
@@ -5,11 +5,11 @@ This MDS data types page catalogs the objects (fields, types, requirements, desc
 ## Table of Contents
 
 - [Vehicles](#vehicles)
-  - [Vehicle Types](#vehicle-types)
```
Comment (schnuerle) [inline data-types.md:8]: This link should remain as an anchor with the #vehicle-types link. It doesn't need to be a shortcut, as it's in a Table of Contents.

File: data-types.md [hunk 3]
```diff
@@ -294,18 +294,18 @@ Other special group types may be added in future MDS releases as relevant agency
...
 [gbfs-station-info]: https://github.com/NABSA/gbfs/blob/master/gbfs.md#station_informationjson
 [gbfs-station-status]: https://github.com/NABSA/gbfs/blob/master/gbfs.md#station_statusjson
 [geography-driven-events]: /general-information.md#geography-driven-events
 [gps]: #gps-data
 [iso4217]: https://en.wikipedia.org/wiki/ISO_4217#Active_codes
 [modes]: /modes/README.md
-[propulsion-types]: /general-information.md#propulsion-types
-[stop-based-geo]: #stop-based-geographic-data
+[propulsion-types]: #propulsion-types
+[stop-based-geo]: /general-information.md#stop-based-geographic-data
```
Comment (schnuerle) [inline data-types.md:305]: I think this should link to /general-information.md#geographic-data as there is no anchor on the page for #stop-based-geographic-data

Stop Based Geographic data was removed in 2.0, in favor of mentioning it in Telemetry:  https://github.com/openmobilityfoundation/mobility-data-specification/blob/25a87400959eee89e2b06b694ef157a5fef8c138/data-types.md#telemetry

Comment (bGrass) [reply data-types.md:305]: stop-based-geo is unused once I repoint the reference on line 111 to Telemetry.

**Comment by bGrass** [comment]
I'm sorry for the slow response, this had slipped off my radar.

**Pull request merged**